falcon/response.py

```diff
@@ -132,6 +132,8 @@ class Response(object):
                 the Response instance itself (self).
 
         options (dict): Set of global options passed from the API handler.
+
+        headers (dict): Read-only list of all headers set for Response.
```

Comment:
Nitpick: We probably should say "dict" or "collection" or something, instead of "list".

Reply:
Would you mind adding a note about the header names being lowercased for performance reasons? People will need to know that we don't use a case-insensitive dict (again, for performance reasons).